src/run.rs

```diff
@@ -256,20 +256,17 @@ pub fn start_subprocesses(config: &Config, tmp_dir: &Path) -> Vec<Child> {
 
 pub fn clean_up(
     delete_file: &Path,
-    tmp_dir: TempDir,
+    tmp_dir: &Path,
     worktree_branch: Option<&str>,
     subprocesses: &mut [Child],
 ) -> Result<()> {
     if !delete_file.exists() {
-        println!(
-            "Project directory preserved at: {}",
-            tmp_dir.into_path().display()
```

Comment:
You broke the logic :grin: https://docs.rs/tempfile/latest/tempfile/struct.TempDir.html#method.into_path